launcher/constants.py

```diff
@@ -60,10 +64,24 @@
             }
         ],
     },
+    "kiwi_install_test_efi": {
+        "description": "Runs the installation from a kiwi disk image in EFI mode",
+        "settings": [
+            {
+                "key": "PUBLISH_HDD_1",
+                "value": "%DISTRI%-%VERSION%-%ARCH%-%BUILD%-efi.qcow2",
```

Comment:
this step forward but still there is a high chance that you will conflict with some other jobs publishing hdd's ( not kiwi related ) . let's do it more unique 
```suggestion
                "value": "kiwi-functional-test-%DISTRI%-%VERSION%-%ARCH%-%BUILD%-efi.qcow2",
```

Reply:
Thanks for the suggestion! I've added the `kiwi` prefix to all published HDDs